### ISSUE TITLE
Feature Request: A way to randomise the colour when painting bricks

### ISSUE THREAD

**Issue by SnowShughes** [guillaumechereau/goxel#164]
Painting imperfections by hand is quite time-consuming. Would be awesome if you could randomise the paintbrushes hue, saturation or lightness on each click so that you could more easily create noise on a surface. Alternatively, a pattern brush would be great where you could select a bunch of different voxels and then paint them as a group on each subsequent click. This would again introduce some randomness to the painting process and greatly speed up having to do this one voxel at a time.

**Comment by guillaumechereau** [comment]
Yes that would be very nice.   I think the best would be to be able to create programmable brushes using either lua or javascript.
